components/02-molecules/meta/event-meta/yds-event-meta-localist.twig

```diff
@@ -168,8 +180,6 @@
             <ul {{ bem('multiple-dates', [], event_meta__base_class) }} aria-expanded="false">
               {% for event_date in event_dates %}
                 <li {{ bem('multiple-dates__date', [], event_meta__base_class) }}>
-                  {{ event_date.formatted_start_date }} {{ event_date.formatted_start_time }} to {{ event_date.formatted_end_date }} {{ event_date.formatted_end_time }}
```

Comment:
So while you deleted your code in your latest push, you didn't revert to the old code that was there. ☹️ You'll want to place back the above that's in red if the green wasn't meant to be there.  See the last resolved version above to see that change.

Reply:
So I have to put the original code back push to git and then fix it?

Reply:
Either the original code back or make appropriate changes that should go there instead.  You can do it in sequential order, too.  Like first restore the original code, then make another commit with your new change.  That one is up to you.  :D

Reply:
I already put the original code on line 183 back. Didnt make any new changes on 183 until issue resolve.